src/Symfony/Component/Form/Tests/Extension/Core/Type/EnumTypeTest.php

```diff
@@ -257,6 +260,24 @@ public function testChoiceLabel()
         $this->assertSame('Yes', $view->children[0]->vars['label']);
     }
 
+    public function testChoiceLabelTranslatable()
+    {
+        $form = $this->factory->create($this->getTestedType(), null, [
+            'multiple' => false,
+            'expanded' => true,
+            'class' => TranslatableTextAlign::class,
+        ]);
+
+        $view = $form->createView();
+
+        $this->assertInstanceOf(TranslatableInterface::class, $view->children[0]->vars['label']);
+
+        $translator = $this->createMock(TranslatorInterface::class);
+        $translator->expects($this->once())->method('trans');
+
+        $view->children[0]->vars['label']->trans($translator);
```

Comment:
Let's check the returned value shall we?

Reply:
Done